src/common/typedefs/ipcBridge.d.ts

```diff
@@ -156,7 +156,7 @@ interface AsyncIPCFunctions {
     shoudlRemoveSetting: boolean
   ) => Promise<void>
   repair: (appName: string, runner: Runner) => Promise<void>
-  moveInstall: (args: MoveGameArgs) => StatusPromise
+  moveInstall: (args: MoveGameArgs) => Promise<void>
```

Comment:
We only call `moveInstall` in one place and we don't really use it's returned value